### PR TITLE
Bump timeouts in build-and-lint’s

### PR DESCRIPTION
**Pull request by cocreature**
We’ve seen some timeouts on CI so let’s bump everything to 100ms and
see if that fixes it. If it doesn’t, we can increase those 100ms as well.

changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by garyverhaegen-da — APPROVED** [review]
Thanks!

**Comment by garyverhaegen-da** [comment]
Looks like 100ms is not enough for the open event. I suppose it as to wait for the JSON API to be able to respond there.

File: language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts [hunk 1]
```diff
@@ -461,24 +461,24 @@ test("multi-{key,query} stream", async () => {
 });
 
 test('stream close behaviour', async () => {
+  const waitTime = 100;
   const url = 'ws' + httpBaseUrl().slice(4) + 'v1/stream/query';
   const events: string[] = [];
   events.push('before creation');
   const ws = new WebSocket(url, ['jwt.token.' + ALICE_TOKEN, 'daml.ws.auth']);
   events.push('after creation');
-  await sleep(10);
```
Comment (cocreature) [inline language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts:469]: In case anyone wonders, with this we can end up registering the `open` event after it has already fired in which case the handler will never get called. Without this, we never yield control so this is guaranteed to not be racy.

**Comment by garyverhaegen-da** [comment]
As discussed on Slack, the issue here is not actually that we don't wait long enough, but a faulty assumption in how the tests were written to begin with: the assumption was that the `open` handler would be called even if registered _after_ the connection is registered, which is actually not the case: it only gets called if it gets registered before the state of the ws object changes. The state of the ws object is changed asynchronously (though still in js), so the guarantee is actually that you'll get the `open` event only if you register the handler in the same JS "thread" as the creation of the connection.

In other words, the bug is the `await` on line 469, not the sleep duration: we're only guaranteed to see the open event if the handler is registered _before_ we relinquish control with an `await`.

**Comment by cocreature** [comment]
The initial failure I saw was a missing `close` event so there _is_ an issue with timeouts as well I believe.
